src/commands/config-set.js

```diff
@@ -0,0 +1,36 @@
+const {Command, flags} = require('@oclif/command')
+const fs = require('fs-extra')
+const path = require('path')
+
+class ConfigSetCommand extends Command {
+  static args = [
+    {name: 'key'},
+    {name: 'value'}
+  ]
+
+  async run() {
+    const {args} = this.parse(ConfigSetCommand)
+    const configFilePath = path.join(this.config.configDir, 'config.json')
+
+    // Initialize config file if it does not yet exit.
+    if (!fs.existsSync(configFilePath)) {
```

Comment:
This gives me an error message in the console when the file does not exist but it succeeds in creating it and filling the values. 

```(node:45493) [ENOENT] Error Plugin: adapt-dev-utils: /Users/admin/.config/adapt-dev-utils/config.json: ENOENT: no such file or directory, open '/Users/admin/.config/adapt-dev-utils/config.json'```